openAPI/core-back-external.yaml

```diff
@@ -50,19 +50,35 @@ paths:
             application/json:
               schema:
                 type: "object"
+        400:
+          description: 400 response
+          content: {}
+        500:
+          description: 500 response
+          content: {}
       x-amazon-apigateway-integration:
-        type: "MOCK"
-        requestTemplates:
-          application/json: "{\"statusCode\":200}"
+        type: aws
+        credentials:
+          Fn::GetAtt: ExternalApiGatewayJwksS3Role.Arn
+        httpMethod: GET
+        uri:
+          Fn::Sub:
+            - arn:aws:apigateway:eu-west-2:s3:path/ipv-core-well-known-jwks-${env}/well-known.json
+            - env:
+                Fn::If:
+                  - IsDevelopment
+                  - Fn::If:
+                      - IsDev01
+                      - dev01
+                      - dev02
+                  - Ref: Environment
         responses:
-          200:
+          default:
             statusCode: 200
-            responseTemplates:
-              application/json:
-                Fn::FindInMap:
-                  - EnvironmentConfiguration
-                  - Ref: AWS::AccountId
-                  - jwksJson
+          4\d{2}:
+            statusCode: 400
```

Comment:
Are you sure this is right? If we get (e.g.) a 4XX from S3 that's very likely a problem on our side (either bad permissions or a missing file), so should probably be a 500 to the client.

Reply:
Good shout, I'll fix it up